src/components/HealthClimateComponents/BlackCarbon.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { Grid, Typography } from '@material-ui/core';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = theme => ({
+  grid: {
+    height: '45rem'
+  },
+  mainTitle: {
+    color: theme.palette.primary.light,
+    textAlign: 'center',
+    fontWeight: 'bolder',
+    textDecoration: 'Upppercase',
+    fontSize: theme.typography.fontSize
+  },
+  textGrid: { paddingBottom: '3rem' },
+  bodyCopy: { textAlign: 'center', marginRight: '0.5rem', marginLeft: '0.5rem' }
+});
+
+function BlackCarbonComponent({ classes }) {
+  return (
+    <Grid
+      item
```

Comment:
At the top level of a component, why are you starting with `<Grid item`? I believe `<Grid container` should always come first and `<Grid item` can only be a child of the said `container`. If you start with `item` then you're automatically assuming this component **WILL** only ever live inside a parent component with `<Grid container`.